kubernetes/samples/scripts/create-rcu-schema/README.md

```diff
@@ -6,50 +6,19 @@ The sample scripts in this directory demonstrate how to:
 * Create an RCU schema in the Oracle DB that will be used by a Fusion Middleware domain.
 * Delete the RCU schema in the Oracle DB used by a Fusion Middleware domain.
 
-## Start an Oracle database service in a Kubernetes cluster
-
-Use this script to create an Oracle DB service in the default namespace, with the default credentials, in the Oracle Database Slim image.
 
-The script assumes that either the image, `container-registry.oracle.com/database/enterprise:12.2.0.1-slim`, is available in the Docker repository, or an `ImagePullSecret` is created for `container-registry.oracle.com`. To create a secret for accessing `container-registry.oracle.com`, see the script `create-image-pull-secret.sh`.
-
-```
+## Start an Oracle database service in a Kubernetes cluster
```

Comment:
1) In multiple places database -> Database
2) Please check the output snippets mentioned in this README. Formatting looks gone wrong somehow. 
All most all the output mentioned in this README looks like a paragraph. Please review.

Reply:
Modified  database to Database